Source/Core/DolphinQt/Debugger/BreakpointWidget.cpp

```diff
@@ -317,48 +338,41 @@ void BreakpointWidget::OnContextMenu()
   }
 
   const auto& selected_item = selected_items.constFirst();
-  const auto bp_address = static_cast<u32>(selected_item->data(Qt::UserRole).toUInt());
-
-  auto is_memory_breakpoint = false;
+  const auto bp_address = static_cast<u32>(selected_item->data(ADDRESS_ROLE).toUInt());
+  auto is_memory_breakpoint = selected_item->data(IS_MEMCHECK_ROLE).toBool();
```

Comment:
Why is this randomly not const?
```suggestion
  const auto is_memory_breakpoint = selected_item->data(IS_MEMCHECK_ROLE).toBool();
```